Added Chocolatey package manager for Windows. (#724)
signalfx/splunk-otel-collector: docs/getting-started/windows-manual.md

The collector logs and errors can be viewed in the Windows Event Viewer.

### Chocolatey installation

#### Package Parameters
The following package parameters are available:

> - `/SPLUNK_ACCESS_TOKEN`: The splunk access token (org token) used to send metric data to OTEL. If not specified default is to `1234`

Reviewer: ```suggestion
- `/SPLUNK_ACCESS_TOKEN`: The Splunk access token (org token) used to send metric data to Splunk Observability Suite.
```

Defaulting to an invalid token is a little unintuitive and sets the user up for issues out of the gate.  I think the Smart Agent doesn't start the service after installation if this is the case but I wonder if we should just make it required.

Reviewer 2: I have made changes for making `SPLUNK_ACCESS_TOKEN` required to start service and removing default invalid value.

- `/SPLUNK_REALM`: The parameter will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_REALM` file. If not specified default is ("us0").

> - `/SPLUNK_INGEST_URL:`: URL of the Splunk ingest  (e.g. `https://ingest.YOUR_SPLUNK_REALM.signalfx.com`). Default value is `https://ingest.us0.signalfx.com`.

Reviewer: `s/YOUR_SPLUNK_REALM/SPLUNK_REALM` throughout?

Reviewer 2: changed `YOUR_SPLUNK_REALM` to `SPLUNK_REALM` throughout.

Reviewer: I'm not sure you've pushed all these changes (still see some remaining)

- `/SPLUNK_API_URL`: URL of the API endpoint (e.g. `https://api.YOUR_SPLUNK_REALM.signalfx.com`). Default value is `https://api.us0.signalfx.com`.

> - `/install_dir`: Installation directory. Defaults to `\Program Files\Splunk\OpenTelemetry Collector`.

Reviewer 3: Let's remove all references to `install_dir` since the installation path is not currently configurable.

Reviewer 2: Removed.

- `/SPLUNK_HEC_URL`: URL of Splunk HEC (e.g. `https://ingest.$SPLUNK_REALM.signalfx.com/v1/log`). Default value is `https://ingest.us0.signalfx.com/v1/log`
- `/SPLUNK_TRACE_URL`: Trace url is end point where apllication traces will be collected. URL of Splunk TRACE (e.g. `https://ingest.$SPLUNK_REALM.signalfx.com/v2/trace`). Default value is `https://ingest.us0.signalfx.com/v2/trace`
- `/SPLUNK_BUNDLE_DIR`: The path to the Agent bundle directory. The default path is provided by the collector package. If the specified path is changed from the default value, the path should be an existing directory on the node. The SPLUNK_BUNDLE_DIR environment variable will be set to this value for the collector service.
- `/MODE`: The mode option is used for setting config_path to `\ProgramData\Splunk\OpenTelemetry Collector\agent_config.yaml` or `\ProgramData\Splunk\OpenTelemetry Collector\gateway_config.yaml` in OpenTelemetry Collector. Possible values are `agent` and `gateway`. Default value is `agent`.

To pass parameters, use `--params "''"` :
```sh

> PS> choco install splunk-otel-collector [choco options] --params="'/SPLUNK_ACCESS_TOKEN:YOUR_SPLUNK_ACCESS_TOKEN /SPLUNK_INGEST_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com /SPLUNK_API_URL:https://api.YOUR_SPLUNK_REALM.signalfx.com /install_dir:YOUR_INSTALL_DIR /SPLUNK_HEC_TOKEN:YOUR_SPLUNK_HEC_TOKEN /SPLUNK_HEC_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com/v1/log /SPLUNK_TRACE_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com/v2/trace /SPLUNK_REALM:YOUR_SPLUNK_REALM /SPLUNK_BUNDLE_DIR:YOUR_DIR_FOR_AGENT_BUNDLE /$MODE:YOUR_MODE_FOR_INSTALLATION'".

Reviewer: I think it'd be preferable for the simplest functional installation with just token and realm.

Reviewer 2: Updated it with only `SPLUNK_ACCESS_TOKEN` and `SPLUNK_REALM`.
